meeting_notes/20160418_meeting-notes.md

```diff
@@ -0,0 +1,155 @@
+2. Roadmap
+3. Breakout Groups
+    - Website Content
+    - Maker Proposal
+
+# Notes
+
+- We will use voting in this meeting: 
+    - _Thumbs up_: go ahead, care about the issue and happy with direction
+    - _Closed fist_: "no-block"
+    - _Thumbs down_: needs more discussion, feel strongly enough to block
+
+## Intros
+
+- Went around and everyone gave a brief intro
+    - If people would like to add in their name for the meeting minutes they were asked to add it to the Meeting Agenda Pad
+- Status Update, current approach been two-pronged:
+    - Vision and establishing a direction for the project
+    - Prototyping hardware
+- Want for an open process, documenting as we go, to be available for people looking up information
+- Think about preliminary outreach to other Mesh organizations (Seattle, NYC, Detroit)
+
+## Vision
+
+- Looking at the problem definition doc: https://docs.google.com/document/d/1XQiqvkuFiTmaVtZcjmwBUfH5cwZdaYYLXHPqWAB1sB0/edit
+- "Internet infrastructure = invisible": Agency, Autonomy, Tech literacy
+- Resiliencly (new): In case something were to happen (e.g. Turkey site blocking)
+    - Redudancy, decenteralized systems over single points of failure, Positive, negative attributes: latency, potentially
+- *Internet iInfrastructure is invisible to most of its users, resulting in end users not having:*
+    - End user: internet is a utility
+    - Some will see infrastrcuture as a barrier of entry
+    - Root problem: People are inclined to not understand how the internet works
+        - Part of Reason: Large technical step
+    - Self-selection bias
+    - How to make avenues for understanding, and approaching infrastructure
+- Q: Are we going for something completely transparent?
+    - Show something that is not legally binding, friendly safe
+    - Accountability: Known risks
+    - How do we build a sustainable community
+    - Ref: Sao Paulo air quality network
+- *High speed access*
+    - Not priority
+    - Ref: http://www.thestar.com/business/tech_news/2016/04/17/what-became-of-torontos-push-for-free-public-wi-fi.html
+        - Paid for by BIA (in most cases, downtown London)
+        - Possible matching funds
+        - Montreal: Not for profit wireless group, part funded by the city
+        - Involve City Council
+        - Wireless Toronto: Funding is possible (needs more investigating)
+        - **Are people are most interested in free public access (vote)**
+            - Not the main thing; Not the first consideration: Are they mutually exclusive
+        - Free wireless diagram: http://networkcultures.org/unlikeus/wp-content/uploads/sites/2/2013/03/networktypes.png
+            - BIA could provide an uplink to the Internet
+        - Is this project mutually exclusive? No
+            - The mesh can hook into an established, provided for (BIA), infrastructure
+        - **Vote: Main concern is not access (majority voted yes)**
```

Comment:
We all has consensus about everything! Maybe we don't say 'majority,' we just way we voted?